veg2hab/mozaiek.py

```diff
@@ -548,6 +548,10 @@ def expand_habkeuze_vegtypeinfo_columns(
         )
 
     result = augmented_overlayed.apply(expand_habkeuze_vegtypeinfo_columns, axis=1)
+
+    if len(result) == 0:
+        return None
+
```

Comment:
Bij een mozaiekvlak zonder buren liep dit fout

Reply:
Is dit beter dan een dataframe met 0 regels?

Reply:
Heb even getest, met een lege dataframe gaat hij in `_check_mozaiekregels()` nog steeds de `for row in self.gdf.itertuples():` in. Dus als ik toch expliciet moet checken of hij leeg is, vind ik dit wel extra lekker duidelijk.